File: problemsKylePade/problemsKylePade_75/solution.ts
/**
 Do not return anything, modify nums in-place instead.
 */
function sortColors(nums: number[]): void {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const nums: number[] = JSON.parse(inputValues[0]);
	sortColors(nums)
	return nums;
}

File: problemsKylePade/problemsKylePade_75/solution.go
package problem75

import (
	"encoding/json"
	"log"
	"strings"
)

func sortColors(nums []int)  {
    
}

func Solve(inputJsonValues string) interface{} {
	inputValues := strings.Split(inputJsonValues, "\n")
	var nums []int

	if err := json.Unmarshal([]byte(inputValues[0]), &nums); err != nil {
		log.Fatal(err)
	}

	sortColors(nums)
	return nums
}
